guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,196 @@
 {
```

Comment:
<blockquote><br></blockquote><blockquote>Adds&nbsp;<code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#pointcnn" target="_blank">PointCNN</a></code>&nbsp;model to segment point clouds into classes</blockquote><p>PointCNN model is not going into 1.7.1 release according to me.</p><blockquote>Adds support for multi-channel data augmentation for training on multispectral imagery</blockquote><p>We support multi-channel data augmentation using fastai and the changes that Sandeep had made to the api have been rolled back.</p><blockquote class="ql-indent-1">set&nbsp;<code>lighting_transforms</code>&nbsp;parameter in&nbsp;<code><a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/583/files/" target="_blank">prepare_data()</a></code>&nbsp;to&nbsp;<code>True</code>&nbsp;or&nbsp;<code>False</code>&nbsp;to control data augmentation</blockquote><p class="ql-indent-1">I need to confirm whether such a parameter is supported in prepare_data function.</p><blockquote class="ql-indent-1">turn on&nbsp;<code>DRA</code>&nbsp;with the&nbsp;<code>statistics_type</code>&nbsp;parameter in various&nbsp;<code>show_results()</code>&nbsp;and&nbsp;<code>show_batch()</code>&nbsp;functions</blockquote><p class="ql-indent-1">This functionality is only supported for multispectral data</p><blockquote class="ql-indent-1">adds environment variable&nbsp;<code>type_init_tail_parameters</code>&nbsp;to control&nbsp;<code>Model Tail</code>&nbsp;initialization for&nbsp;<code>arcgis.learn</code>&nbsp;functions</blockquote><p class="ql-indent-1">This one is also for multispectral purpose only.</p><blockquote><br></blockquote><blockquote>Enhances module to create pre-trained backbones on satellite imagery</blockquote><blockquote class="ql-indent-1">use custom pretrained backbones</blockquote><blockquote class="ql-indent-1">Esri World Imagery</blockquote><blockquote class="ql-indent-1">landsat imagery</blockquote><blockquote class="ql-indent-1">sentinel imagery</blockquote><p class="ql-indent-1">It is not yet supported for v1.7.1 release.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<ul><li>I removed the PiontCNN, multi-channel data augmentation, <code>lighting_transforms</code> (I don't see it in the current build api ref), and backbone notes</li><li>I edites the <code>DRA</code> and <code>type_init_tail_parameters</code> to look like this:</li></ul><p>                   <em>Enhancements to Multispectral data support:&nbsp;</em></p><ul><li class="ql-indent-1"><em>turn on <code>DRA</code> with the <code>statistics_type</code> parameter in various <code>show_results()</code> and <code>show_batch()</code> functions</em></li><li class="ql-indent-1"><em>adds environment variable <code>type_init_tail_parameters</code> to control <code>Model Tail</code> initialization for <code>arcgis.learn</code> functions</em></li></ul>